server/metrics/fdb.go

```diff
@@ -26,17 +26,59 @@ var (
 	FdbRespTime      tally.Scope
 )
 
+func getFdbOkTagKeys() []string {
```

Comment:
What do you think about moving all the tag helper functions to a separate file in the same package, like `tags.go`?

Reply:
I think they belong to the metric group. When you create a new group of metric, you create one of these files, which has the initialisation methods, the scope, and the tags the metrics will have. What do you think?